packages/node_modules/@webex/plugin-meetings/sample/app.js

```diff
@@ -493,18 +510,59 @@ document.getElementById('join').addEventListener('click', () => {
       handleJoinIntent(err, meeting, deviceId);
     });
 
-  meeting.participants.on('participants:created', (participants) => {
-    document.getElementById('participants-collection').innerHTML = `PARTICIPANTS INFO: 
-    selfList-${getParticipantString(participants.selfList)}-
-    hostList-${getParticipantString(participants.hostList)}-
-    inLobbyList-${getParticipantString(participants.inLobbyList)}-
-    inMeetingList-${getParticipantString(participants.inMeetingList)}-
-    notInMeetingList-${getParticipantString(participants.notInMeetingList)}-
+  meeting.on('meeting:self:lobbyWaiting', () => {
+    document.getElementById('lobby-space').innerHTML = 'User is guest to space, waiting to be admitted, wait to use addMedia';
+  });
+
+  meeting.on('meeting:self:guestAdmitted', () => {
```

Comment:
Do we need the `self` distinction here? Do we have an equivalent `meeting:remote:guestAdmitted`? Trying to understand if the end user will really care _who_ did it and if they will need to listen for multiple events but end up taking the same action in the UI.